docs/cli_commands.md

```diff
@@ -57,6 +57,12 @@ Must be in keyboard directory with a default keymap, or in keymap directory for
 qmk compile
 ```
 
+**Usage for building all user keymaps**:
```

Comment:
Maybe it's just me, but to me this means "build the keymaps of all users".

Otherwise, LGTM.